src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -380,7 +380,7 @@ class CellInfoContentProvider {
 			return existing;
 		}
 
-		const data = CellUri.parseCellUri(resource, Schemas.vscodeNotebookCellMetadata);
```

Comment:
Were the changes in core included by mistake?

Reply:
@mjbvz right, it's an oversight introduced via Cell fragment change.